stdlib/private/OSLog/StringAlignment.swift

```diff
@@ -0,0 +1,67 @@
+@frozen
+public enum CollectionBound {
+  case start
+  case end
+}
+
+extension String {
+  @frozen
+  public struct Alignment {
```

Comment:
I'd pick a specific name to avoid potential future conflicts or shadowing like OSLogStringAlignment, and not have this be in the String namespace